conf.pri.windows

```diff
@@ -13,17 +13,18 @@ LIBS += $$quote(-LC:/qBittorrent/openssl/lib)
 # Adapt the lib names/versions accordingly
 # If you want to use Boost auto-linking then disable
 # BOOST_ALL_NO_LIB below and omit Boost libraries here
+LIBS += zlib.lib libcrypto.lib libssl.lib
```

Comment:
please move it back, it is intended being there.

Reply:
Or we can maintain the order by lib names in each section, i.e. with the following order: [boost, libtorrent, openssl, zlib, qbt specifics].

Reply:
ZLIB also carries a debug suffix on windows (-> zlibd.lib)

Reply:
Done